src/options.js

```diff
@@ -1,6 +1,6 @@
 const SHOW_SIDEBAR_OPTION = {
     elementId: 'switch-sidebar',
-    defaultValue: true,
+    defaultValue: false,

```

Comment:
I don't think we want to change this default parameter.

```suggestion
    defaultValue: true,
```